tox.ini

```diff
@@ -1,24 +1,21 @@
 [tox]
-passenv = 3.6.0
-envlist = python2.7, python3.6, python3.7, python3.8, flake8
-skip_missing_interpreters=True
+envlist = py2.7, py3.6, py3.7, py3.8, flake8
+skip_missing_interpreters=true
 
 [testenv:flake8]
 basepython = python
 deps = flake8
-commands = flake8 --max-line-length=100 eule tests
+commands = flake8 --max-line-length=100 src tests
```

Comment:
As in your `setup.py` you used the directive to search for the source code in the `src` directory, I both moved it there and I updated that command.